<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabela">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>841</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDados">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dados</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Busca Sequencial - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busca Sequencial - Matricula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busca Binaria - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Busca Binaria - Matricula</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonAbrir">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAtt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenacao</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOrdenacao">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string> Nenhum</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> Matricula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> Departamento - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> Titulação - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> Tipo de Contrato - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> Departamento - Titulação - Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> Departamento - Tipo de Contrato - Nome</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonAttOK">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResult">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
